app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABCMeta, abstractmethod
+
+from app.players.player import Player
+
+
+# metaclass=ABCMeta makes IDE recognize that this is also an abstract class.
+class Dwarf(Player, metaclass=ABCMeta):
+    @abstractmethod
```

Comment:
why did you make `__init__` an abstract method?

Reply:
btw, you can inherit this class from `ABC` and avoid using `metaclass`

Reply:
I thought `__init__` should be abstract because it was required to implement in all the subclasses. Can you please clarify why it shouldn't?